bitnami/mongodb/templates/standalone/svc.yaml

```diff
@@ -50,8 +50,6 @@ spec:
       targetPort: mongodb
       {{- if and (or (eq .Values.service.type "LoadBalancer") (eq .Values.service.type "NodePort")) .Values.service.nodePorts.mongodb }}
       nodePort: {{ .Values.service.nodePorts.mongodb }}
-      {{- else if eq .Values.service.type "ClusterIP" }}
-      nodePort: null
```

Comment:
I think it doesn't need to remove these lines. 

> fixes a bug where node port was rendered with null instead of not being rendered. nodePort is an integer and it can't be null.

NodePort is null in case the type is CluterIP.

Reply:
It's a warning there is no issue. If you check, it is in a lot of the charts.

Reply:
Got Ya, what about the other change with the subpath ?

Reply:
Hey, any update